### DISCUSSION TITLE
Tubular secretion transporters

### DISCUSSION THREAD

**Discussion by bahsa505** [Open-Systems-Pharmacology/Forum#1200] [Questions & Problems]
Dear all,

I want to know how do we specify the renal transporters (basolateral, apical) involved in tubular secretion and how would this differ from implementing the tubular section option in renal clearance? 

![image](https://user-images.githubusercontent.com/61779799/163971249-8d72311d-6794-4941-a802-981ff6db25bb.png)

If we are to use the tubular secretion option, only one process kinetics is required to be provided, but what does this indicate? is this kinetics of drug transport from the tubular cells to the urine? if this is the case would it be equivalent to defining the apical transporter kinetics?

Also, if I have the information on the tubular uptake transporter involved in secretion (e.g. OAT3), but limited knowledge of the efflux transporter and I want to define it as a hypothetical efflux transporter with values for Vmax, km.. how would this be implemented given that we don't have expression data (we want to keep it exclusively expressed in the kidney in this case)?
 
![image](https://user-images.githubusercontent.com/61779799/163970115-96810c35-3a61-402c-aafe-21fc3a632015.png)

Kind regards
Salma

**Comment by prvmalik** [comment]
Insert two separate transporters in the ADME tab under "Transport Proteins." One influx transporter that is expressed on the basolateral membrane (e.g. OAT1 or OAT3). One efflux transporter that is expressed on the apical membrane. Define Vmax and Km individually.

However there is often no way to identify the Vmax of each of these transporters through optimization and there is no way to parameterize reliably from in vitro in any PBPK modeling system to date. An in vitro Km has fairly good value in translation. 

One way to identify an OAT Vmax separately from the apical transporter Vmax is by optimizing against an observed DDI with probenecid.

If there is no information that would allow separate identification of both the uptake and efflux processes, then an assumption that the apical efflux is rate limiting could be an appropriate work around. In this case only one net transporter is required. 

I believe the tubular secretion option under the renal clearance process eliminates drug directly from plasma to urine with a Michaelis Menten approximation.

**Reply by bahsa505** [reply]
Thanks a lot for your input and insights about the identifiability of the relevant parameters. I am actually trying to reproduce a published model where both the transporters were considered.

I have tried defining the transporters at the compound ADME and allowing the simulation individual to have the expression of these transporters. I was wondering though in the case where one of the transporter's expression (OAT in this case) is informed by the expression library, but not the other. I assume that in the context of renal secretion the kidney is the organ of interest, so how can we accommodate for the fact that the known transporter could be expressed in other organs as well? would manual addition of the known transporter (keeping the expression in kidney only) be a valid option? Also is there a way to consider the reported variabililty in Vmax,km in the literature?

**Reply by prvmalik** [reply]
Yes each transporter is assigned uniquely. One can be from the expression database and one could be a 'default' or 'dummy' transporter with manual expression. 

If you link the published paper I can probably explain what they did.

**Reply by prvmalik** [reply]
Variability in Vmax and Km can be handled with sensitivity analysis and/or optimization, if identifiable. 

Say you have multiple Km values in literature (1, 2, 5) due to experimental variation. You could parameterize your model with the median or mean. To test whether your assumption has an impact on model outcomes, you could test model outcomes when the most uncertain values are used instead (the highest and lowest) in a process known as sensitivity analysis.

Inter-subject variability is handled differently. We use populations and apply intersubject variability to the transporter reference concentration, which typically includes indirect intersubject variability on the Vmax. Vmax for the ODEs in the model is typically calculated per umol of enzyme.

**Reply by bahsa505** [reply]
Thanks, Paul. It is this model

https://link.springer.com/article/10.1007/s40262-021-01046-6#Sec22

I found that the simulation results are greatly affected by the transporter expression library source since the transporter of interest has quite different expression profiles among the different library sources. The closest predictions are the ones based on limited transporter expression in the kidney.

For the variability of the transporter kinetics, since I am reproducing the published simulation, intersubject/between studies variability is what I am trying to account for here. I see in the documentation of user-defined variability as you mentioned that the variation of the reference concentration of the enzyme is what we can implement. I understand how this could be linked to the Vmax variability. It is a bit unclear though how we can translate the variability of both Vmax, and km on the reference concentration? I wonder if there is documentation on this?

**Reply by prvmalik** [reply]
First this is done in PKSim 8 and the basolateral OAT transporter kinetics cannot be translated exactly to PK10+.

Two transporters have been inserted using the "Specific Active Transporter - Michaelis Menten" process under the "Transport Proteins" in the Meropenem ADME tab. There is OAT3 influx (1 uM) on the basolateral side of the kidney and a dummy transporter efflux (1 uM) on the apical side of the kidney. Vmax and Km values for each transporter are in this table. It does not explicitly state, but OAT3 is likely uploaded with the expression profile of RT-PCR from the Human Gene Expression Database in PK-Sim.

To my knowledge no PBPK model has ever applied variability on a KM parameter.

![image](https://user-images.githubusercontent.com/29066045/164757429-c553a7fb-5a3d-48eb-ab08-b082c3d2d73b.png)

**Comment by WTY-tt** [comment]
Hello, I am also using PK-SIM for modeling recently, which involves OAT3. The Vmax unit I searched is pmol/min/mg, and I don't know how to convert it into µmol/L/min in PK-SIM. I would like to ask how to convert the Vmax you finally used into units, is it according to the reference [24]?
